lib/kuzzle/kuzzle.ts

```diff
@@ -299,6 +371,166 @@ class Kuzzle extends KuzzleEventEmitter {
     }
   }
 
+  // For testing purpose
+  async ask (...args: any[]) {
+    return super.ask(...args);
+  }
+
+  // For testing purpose
+  async emit (...args: any[]) {
+    return super.emit(...args);
+  }
+
+  // For testing purpose
+  async pipe (...args: any[]) {
+    return super.pipe(...args);
+  }
+
+  private async importUserMappings (
+    config: {
+      toImport: ImportConfig,
+      toSupport: SupportConfig
+    },
+    status: ImportStatus
+  ): Promise<void> {
+    if (! status.isFirstCall) {
+      return;
+    }
+
+    const toImport = config.toImport;
+
+    if (! _.isEmpty(toImport.userMappings)) {
+      await this.internalIndex.updateMapping('users', toImport.userMappings);
+      await this.internalIndex.refreshCollection('users');
+      this.log.info('[✔] User mappings import successful');
+    }
+  }
+
+  private async importMappings (
+    config: {
+      toImport: ImportConfig,
+      toSupport: SupportConfig
+    }
```

Comment:
Here you are not using the `ImportStatus` object to avoid re-importing mappings, is it intentional ?

Reply:
Oops id did rollback some of my commit by accident, this it not intended